packages/ember-data/tests/integration/adapter/json-api-adapter-test.js

```diff
@@ -875,3 +875,34 @@ test('update record - serialize hasMany', function() {
     });
   });
 });
+
+test('fetching a relationship via link works with data = null - #3451', function() {
```

Comment:
Not so sure about the test name but couldn't come up with a better one. Suggestions?